### PR TITLE
updating Multiple Ascending dose PK Rmd file

### PR DESCRIPTION
**Pull request by margoal1**

### CODE REVIEW

File: Rmarkdown/Multiple_Ascending_Dose_PK.Rmd [hunk 1]
```diff
@@ -1,573 +1,451 @@
...
+
+When looking at summaries of PK over time, there are several things to observe. Note the number of doses and number of time points or sampling schedule. Observe the overall shape of the average profiles. What is the average Cmax per dose? Tmax? Does the elimination phase appear to be parallel across the different doses? Is there separation between the profiles for different doses? Can you make a visual estimate of the number of compartments that would be needed in a PK model?
+
+### Concentration over Time, colored by dose, mean +/- 95% CI
+
+```{r , cache = TRUE, echo = TRUE, warning=FALSE, message=FALSE, fig.width = 10, fig.height = 5}
+
+gg <- ggplot(data = pk_data, 
+             aes(x = NOMTIME, y = LIDV, 
+                 group= interaction(DOSE,CYCLE), color = TRTACT_high2low))
```
Comment (iamstein) [inline Rmarkdown/Multiple_Ascending_Dose_PK.Rmd]: add space between "group" and "=".  I'm maybe too anal...

File: Rmarkdown/Multiple_Ascending_Dose_PK.Rmd [hunk 2]
```diff
@@ -1,573 +1,451 @@
...
+gg <- ggplot(data = NCA %>% subset(PARAM %in% c("AUC_0_24", "AUC_tau")), aes(x = DOSE, y = VALUE_NORM))
+gg <- gg + geom_boxplot(aes(group = DOSE)) + geom_smooth(method = "lm", color = "black")
+gg <- gg + scale_x_continuous(breaks=unique(NCA$DOSE))
+gg <- gg + ylab("Dose normalized AUCtau (h.(ng/mL)/mg)") 
+gg <- gg + xlab("Dose (mg)")
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + xgx_annotate_filenames(dirs)
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width,height,dirs,"filename_main",status)
+print(gg + facet_grid(.~DAY_label))
```
Comment (iamstein) [inline Rmarkdown/Multiple_Ascending_Dose_PK.Rmd]: this is weird.  for consistency, i think it should be gg <- gg + facet_grid(...) further up.  And then print(gg) at the end.

File: Rmarkdown/Multiple_Ascending_Dose_PK.Rmd [hunk 3]
```diff
@@ -1,573 +1,451 @@
...
+gg <- gg + xgx_geom_ci(conf_level = 0.95)
+gg <- gg + xgx_scale_y_log10()
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+gg <- gg + ylab(conc_label)
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + xgx_annotate_filenames(dirs)
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width,height,dirs,"filename_main",status)
+print(gg + facet_grid(.~TRTACT_low2high))
```
Comment (iamstein) [inline Rmarkdown/Multiple_Ascending_Dose_PK.Rmd]: don't need print command

File: Rmarkdown/Multiple_Ascending_Dose_PK.Rmd (hunk 1 above)
Comment (margoal1) [reply Rmarkdown/Multiple_Ascending_Dose_PK.Rmd]: Added spaces around all "="

File: Rmarkdown/Multiple_Ascending_Dose_PK.Rmd (hunk 2 above)
Comment (margoal1) [reply Rmarkdown/Multiple_Ascending_Dose_PK.Rmd]: wanted to reuse gg without facet for the next plot

**Pull request merged**